youtube_dl/extractor/wdr.py

```diff
@@ -239,6 +241,59 @@ def _real_extract(self, url):
         return info_dict
 
 
+class WDRElefantIE(WDRBaseIE):
+    _VALID_URL = r'https?://(?:www\.)wdrmaus.de/elefantenseite/#(?P<display_id>.+)'
+    IE_NAME = 'wdr:elefant'
+
+    _TESTS = [
+        {
+            'url': 'http://www.wdrmaus.de/elefantenseite/#lieder_geburtstagslied',
+            'info_dict': {
+                'title': 'Ich bin schon 1-2-3',
+                'id': 'mdb-1008774',
+                'ext': 'mp4',
+                'age_limit': None,
+                'upload_date': '20091119'
+            },
+        },
+        {
+            'url': 'http://www.wdrmaus.de/elefantenseite/#folge_ostern_2015',
+            'info_dict': {
+                'title': 'Folge Oster-Spezial 2015',
+                'id': 'mdb-1088195',
+                'ext': 'mp4',
+                'age_limit': None,
+                'upload_date': '20150406'
+            },
```

Comment:
Remove duplicates.

Reply:
What do you mean by "Remove duplicates"? What is duplicate here? Sorry, I can't follow.

Reply:
Remove all tests that test the same extraction scenario.

Reply:
Ah, ok. I added two tests to increase the chances of having at least one working test, because the site removes most videos after some time (with the time depending on strange rules). However, if you feel one test is enough, I'll drop the other.
Done.